src/cpuid/src/cpu_leaf.rs

```diff
@@ -295,6 +295,16 @@ pub mod leaf_0x80000001 {
     }
 }
 
+pub mod leaf_0x80000006 {
+    pub const LEAF_NUM: u32 = 0x8000_0006;
+
+    pub mod edx {
+        use crate::bit_helper::BitRange;
+
+        pub const RESERVED_BITRANGE: BitRange = bit_range!(16, 17);
```

Comment:
nit: are there any reasons why using `bit_range!(16, 17)` in commit 651b66b0b2a63feb7e74976adcf8887b8d6003a2 and fixing to `bit_range!(17, 16)` in commit b1e1fdf6efd6a93495c8afddc68f1ce23d774370?

Reply:
Absolutely no reason. Just fixed up a wrong commit. Fixed, thanks.